tools/esdlcmd/esdlcmd_common.hpp

```diff
@@ -300,4 +303,27 @@ class EsdlHelperConvertCmd : public EsdlConvertCmd
     virtual void setTransformParams(IProperties *params )=0;
 };
 
+static bool getCurrentFolder(StringBuffer & path)
```

Comment:
minor: unusual to have static functions defined in header functions.  More normal to be declared inline.